client/src/pages/ErrorPage/ErrorPage.js

```diff
@@ -1,5 +1,45 @@
 import React from 'react';
+import { Box, Typography } from '@mui/material';
+import Button from '@mui/material/Button';
+import { useNavigate } from 'react-router-dom';
+import styled from '@emotion/styled';
 
-const ErrorPage = () => <div>Error</div>;
+const ErrorContainer = styled('dix')({
+  width: '100%',
+  height: '100%',
+  textAlign: 'center',
+  '& > div': {
+    display: 'flex',
+    justifyContent: 'center',
+    '& > img': {
+      width: '50vw',
+      maxWidth: '500px',
+    },
+  },
+  '& > h2': {
+    fontSize: '25px',
+    lineHeight: '30px',
```

Comment:
please delete these. no need to change the default style of our h2 tag.

![image](https://user-images.githubusercontent.com/107735322/199251223-c5120738-706d-4f02-81ef-910848d48b64.png)